packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -78,11 +78,11 @@ spec:
                 3.5.0=quay.io/strimzi/kafka:latest-kafka-3.5.0
                 3.5.1=quay.io/strimzi/kafka:latest-kafka-3.5.1
             - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
-              value: quay.io/strimzi/operator:latest
+              value: quay.io/ppatierno/operator:kraft-anno
```

Comment:
I guess some of these things were not supposed to be committed?

Reply:
You were too fast commenting, I was just reverting back changes :-)